docs/build.sbt

```diff
@@ -0,0 +1,10 @@
+// Uses the out of the box generic theme.
+paradoxTheme := Some(builtinParadoxTheme("generic"))
+
+scalaVersion := "2.12.2"
+
+paradoxProperties in Compile ++= Map(
+  "snip.g8root.base_dir" -> "../../../../src/main/g8",
+  "snip.g8src.base_dir" -> "../../../../src/main/g8/src/main/",
+  "snip.g8srctest.base_dir" -> "../../../../src/main/g8/src/test/"
```

Comment:
You can `sourceDirectory in root`. Since this is a separate `build.sbt` without the root value, you should be able to access root with `LocalProject("root")`. And you can get the `compile:sourceDirectory` for `src/main`. So something like this could work:

```
"snip.g8src.base_dir" -> s"${(sourceDirectory in (LocalProject("root"), Compile)).value}/g8/src/main/"
```